### DISCUSSION TITLE
status URL?

### DISCUSSION THREAD

**Discussion by angelnu** [OpenCCU/OpenCCU#1170] [🇺🇸 Q&A] [answered]
I am adding monitoring of my home infrastructure with https://github.com/statping/statping

It basically relays on regularly calling URLs and checking the response code and potentially content. 

Since my RaspberryMatic is configured with autologing (once I pass my home Single-Log-On) calling the main page of homematic results in a session being created which eventually hit the max number, specially if the http ping is each minute.

So is there a URL to check status of the RaspberryMatic without login and therefore consuming a session ID?

**Comment by jens-maus** [comment, marked as answer]
> Since my RaspberryMatic is configured with autologing (once I pass my home Single-Log-On) calling the main page of homematic results in a session being created which eventually hit the max number, specially if the http ping is each minute.

That's exactly one reason why in a future version we will probably retired the auto-login feature of the WebUI completly. It is not only consuming a session-id each time and thus would be a perfect place for DoS attacks, it also has other security related issues. So my suggestion would be to switch of auto-login and then use the standard http://homematic-raspi/login.htm URL as a status type of URL.

> So is there a URL to check status of the RaspberryMatic without login and therefore consuming a session ID?

The only addition URL you could potentially use is the UPNP service URL usually available at:

```
http://<IP>/upnp/basic_dev.cgi
```

This should return some XML stuff. However, this will also be served even thought ReGaHss or other necessary services are down, thus only means the `lighttpd` webserver is up.

**Reply by angelnu** [reply]
Thanks @jens-maus - I will try to disable auto-login - if it works stable on my setup I would add the check to the Helm Chart
